yarn-project/sequencer-client/src/slasher/index.ts

```diff
@@ -0,0 +1,24 @@
+import type { L2BlockSource } from '@aztec/circuit-types';
+import { type L1ContractsConfig, type L1ReaderConfig } from '@aztec/ethereum';
+import { createLogger } from '@aztec/foundation/log';
+import { type AztecKVStore } from '@aztec/kv-store';
+import { type DataStoreConfig } from '@aztec/kv-store/config';
+import { createStore } from '@aztec/kv-store/lmdb';
+import { type TelemetryClient } from '@aztec/telemetry-client';
+import { NoopTelemetryClient } from '@aztec/telemetry-client/noop';
+
+import { SlasherClient } from './slasher_client.js';
+import { type SlasherConfig } from './slasher_client.js';
+
+export * from './slasher_client.js';
+
+export const createSlasherClient = async (
```

Comment:
code consistency nit: in the rest of the codebase this is usually in a factory.ts

Reply:
The p2p is the same with the index, was based on that 😬 Can create a factory for it though 🤷

Reply:
ah yeah, thats got to change too